site/en/docs/android/custom-tabs/guide-engagement-signals/index.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: "layouts/doc-post.njk"
+title: Measuring user engagement
+seoTitle: "Guide: measuring user engagement in Android Custom Tabs"
+date: 2023-04-21
```

Comment:
On both extensions and Fugu, I've always made it the publication date or very close. On shorter items, the difference is usually a day or two, so I don't think it matters that much. For larger projects, particularly those we announce, I like the announcement and the publication date to be as close as possible. I don't like announcing something as new and showing a date that's a month old. 

To summarize:

* For shore items, it doesn't matter.
* For long items, let's assume it will be announced and use the publication date.